creation/lib/cgWDictFile.py

```diff
@@ -211,6 +239,15 @@ def is_equal(
 
 # internal, do not use from outside the module
 def get_common_dicts(submit_dir, stage_dir):
+    """Return a dictionary with new instances of all the common (on both global and entry) dictionaries
+
+    Args:
+        submit_dir:
+        stage_dir:
```

Comment:
Throughout this file, some parts of the docstrings that were added are empty. I’m assuming that they will be added in the future. Is that the plan?

Reply:
In the future